### PR TITLE
feat(byon): transit gateway enablement

### PR DESCRIPTION
**Pull request by pbabz**
Issue #, if available: GALI-1858

Description of changes:
Add new templates. Filed GALI-1955 as a follow up story

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/src/templates/onboard-account-tgw.cfn.yaml [hunk 1]
```diff
@@ -0,0 +1,777 @@
...
+                  - !Sub 'arn:aws:events:${AWS::Region}:${MainAccountId}:event-bus/default'
+
+
+  GatewayLoadBalancerSubnet1:
+    Condition: PublicVPC
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId: !Ref "VPC"
+      CidrBlock: !Ref "GLBSubnet1CIDR"
+      AvailabilityZone: "us-east-1a"
```
Comment (SanketD92) [inline solutions/swb-reference/src/templates/onboard-account-tgw.cfn.yaml:738]: Can we customize this? Not all SWB deployments will be in this region.

Comment (pbabz) [reply solutions/swb-reference/src/templates/onboard-account-tgw.cfn.yaml:738]: I have making this generic as an AC in the follow up story, but I can do it here if you like

**Review by SanketD92 — DISMISSED** [review]

File: solutions/swb-reference/src/templates/onboard-account-tgw.cfn.yaml (hunk 1 above)
Comment (SanketD92) [reply solutions/swb-reference/src/templates/onboard-account-tgw.cfn.yaml:738]: Nope. Can wait for your upcoming ticket.

Comment (pbabz) [reply solutions/swb-reference/src/templates/onboard-account-tgw.cfn.yaml:738]: Fixed it

**Review by SanketD92 — APPROVED** [review]

**Pull request merged**